Realm/Tests/InterprocessTests.m

```diff
@@ -173,7 +173,7 @@ - (void)testBidirectionalCommunication {
     }
 
     RLMNotificationToken *token = [realm addNotificationBlock:^(__unused NSString *note, __unused RLMRealm *realm) {
-        if (obj.intCol % 2 != self.isParent && obj.intCol < stopValue) {
+        if (obj.intCol % 2 == self.isParent && obj.intCol < stopValue) {
```

Comment:
The previous version didn't match the intent of the test, but happened to work due to a race condition. The new version also passes on master.